.eslintrc.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
     ecmaFeatures: {
       jsx: true,
     },
-    project: ['tsconfig.json', 'cypress/tsconfig.json'],
+    project: ['./tsconfig.json', './cypress/tsconfig.json'],
```

Comment:
This is a subtle change, but it seems that the CLI understands, but the VSCode plugin has an issue